college_flow_app/lib/presentation/walkthrough/widgets/walkthrough_base_page.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:college_flow_app/config/design_system/data/colors/colors.dart';
+import 'package:college_flow_app/config/design_system/data/spacing/spacing.dart';
+import 'package:college_flow_app/presentation/widgets/gap.dart';
+import 'package:flutter/material.dart';
+
+class WalkthroughBasePage extends StatelessWidget {
+  final String title;
+  final String description;
+  final String imgPath;
+
+  const WalkthroughBasePage({
+    Key? key,
+    required this.title,
+    required this.description,
+    required this.imgPath,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        horizontal: spacingNano,
+      ),
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          const VSpacer.lg(),
+          Text(
+            title,
+            style: Theme.of(context).textTheme.displayMedium?.copyWith(
+                  color: white,
+                ),
+          ),
+          const VSpacer.nano(),
+          Text(
+            description,
+            style: Theme.of(context).textTheme.bodyLarge?.copyWith(
+                  color: white,
+                ),
+          ),
+          const VSpacer.lg(),
+          Stack(
+            children: [
+              AspectRatio(
+                aspectRatio: 1,
+                child: Container(
+                  width: 30,
+                  decoration: const BoxDecoration(
+                    shape: BoxShape.circle,
+                    color: primaryLight,
+                  ),
+                ),
+              ),
+              //TODO(Mauricio-Machado): Add image here
+              const SizedBox.shrink(),
```

Comment:
Change this stack for just one image, remove the svgs from the project. Export them from sigma as pngs. @Luanmq can you help him? Any doubts just hmu

Reply:
I fix that, on commits [c52578f](https://github.com/fga-eps-mds/CollegeFlow/pull/104/commits/c52578faffd76d41765850bda637c00797bc6110) and [3a5654c](https://github.com/fga-eps-mds/CollegeFlow/pull/104/commits/3a5654c4a8fee2b1ecdcf88387484f5bdecf9bcd)